### ISSUE TITLE
Add double-licensing (CeCILL-B + Apache V2)

### ISSUE THREAD

**Issue by anarchodin** [pchampin/sophia_rs#164]
In looking for RDF libraries to use from Rust, this package caught my attention. However, I also noticed that it uses a GPL-incompatible licence, which sort of excludes it from consideration for me. Is this by design?

**Comment by pchampin** [comment]
Short answer is "no", it was not by design.
As a French civil servant, I chose a license endorsed by French institution, and I chose the one that was presented as similar to the BSD and Apache  licenses. But apparently, the [FSF considers it to be still incompatible with GPL](https://www.gnu.org/licenses/license-list.html#CeCILL-B), which I didn't know...

If Sophia was double-licensed as CeCILL-B and [Apache V2](https://www.gnu.org/licenses/license-list.html#apache2), would this solve your problem?
